src/models/acopf/acopf_auglag_linelimit_kernel_ka.jl

```diff
@@ -0,0 +1,151 @@
+@kernel function auglag_linelimit_two_level_alternative_ka(
+        N::Val{n}, nline::Int, line_start::Int,
```

Comment:
Why do we have to pass `N` as `Val` here? To use `@localmem`? (genuine question)